woocommerce-gateway-ebanx/includes/class-wc-ebanx-gateway-utils.php

```diff
@@ -40,6 +40,17 @@
 	);
 
 	/**
+	 *  Local currencies that EBANX processes
+	 */
+	public static $LOCAL_CURRENCIES = array(
+		self::CURRENCY_CODE_BRL,
+		self::CURRENCY_CODE_CLP,
+		self::CURRENCY_CODE_COP,
+		self::CURRENCY_CODE_MXN,
+		self::CURRENCY_CODE_EUR
```

Comment:
EUR isn't local currency, EUR has the same behavior of USD. Please, remove it ;)